src/ServiceProvider.php

```diff
@@ -32,6 +32,13 @@ public function boot()
                __DIR__.'/../config/state-machine.php' => config_path('state-machine.php'),
            ], 'config');
         }
+
+        if (! class_exists('CreateStateHistoryTable')) {
+            $timestamp = date('Y_m_d_His', time());
+            $this->publishes([
```

Comment:
Please change the value to `databasePath('migrations/'.date('Y_m_d_His', time()).'_create_state_history_table.php'),`. The `$timestamp` temp variable is not necessary.